src/TwigComponent/doc/index.rst

```diff
@@ -1058,6 +1058,143 @@ Exclude specific attributes:
       My Component!
     </div>
 
+.. __Customize your component:
+
+CVA (Class variant authority)
+-----------------------------
+
+You can use the cva function to define variants for your component.
+
+.. code-block:: html+twig
+
+    {# templates/components/Alert.html.twig #}
+    {% props color = 'blue', size = 'md', class = '' %}
```

Comment:
What's the point of the `class` prop? Is that necessary or can we just let the normal attributes system handle things if `class` is passed in?

Reply:
I think this is still coming from earler, when there was also a tailwind merge baked in, one could just leave it out and use it as normal attribute. 

But maybe someone puts custom tailwind in "class" attribute of the component and expects they are merged with the ones the component sets based on variants, I guess if that is the goal it's required to make class a prop and then pass it as second argument to apply and then use tw_merge from other bundle to clean up classes before actually rendering it in the class attribute of the element inside of the component :) 

Maybe docs could show both as an example?